network.py

```diff
@@ -25,6 +25,7 @@ def change_server(self, new_ip):
         """
         self.networktables.shutdown()
         self.networktables.initialize(server=new_ip)
+        self.table = self.networktables.getTable(self.table_name)
```

Comment:
Are we using this or are we jumping the gun with something?

Reply:
`change_server` is used if you want to change what IP the driver station is connecting to *after* it is started. I forgot to change `self.table` to reference a table on the new NetworkTables network, so that just makes sure the reference is up to date.